File: java/src/main/java/mx/iteso/EndUser.java
package mx.iteso;

public class EndUser extends User{

    public EndUser(String username){
        this.username = username;
    }

    @Override
    public boolean publish(){
        System.out.print("No tiene elprivilegio");
        return false;            
    }

    @Override
    public boolean delete(){
        System.out.print("No tiene elprivilegio");
        return false;        
    }

    @Override
    public boolean aprove(){
        System.out.print("No tiene elprivilegio");
        return false;
    }
    
    @Override
    public boolean comment(){
        System.out.print("No tiene elprivilegio");
        return false;
    }
}

File: python/src/main/python/EndUser.py
from User import *


class EndUser(User):

    def __init__(self, username: str):
        super().__init__(username)

    def publish(self) -> bool:
        print("No tiene el privilegio")
        return False

    def delete(self) -> bool:
        print("No tiene el privilegio")
        return False

    def aprove(self) -> bool:
        print("No tiene el privilegio")
        return False

    def comment(self) -> bool:
        print("No tiene el privilegio")
        return False
